src/mmveri.c

```diff
@@ -22,41 +22,27 @@ struct getStep_struct getStep = {0, 0, 0, 0, 0,
 /* Returns 0 if proof is OK; 1 if proof is incomplete (has '?' tokens);
    returns 2 if error found; returns 3 if not severe error
    found; returns 4 if not a $p statement */
-char verifyProof(long statemNum)
-{
-
-  long stmt; /* Contents of proof string location */
-  long i, j, step;
-  char type;
-  char *fbPtr;
-  long tokenLength;
-  long numReqHyp;
-  char returnFlag = 0;
-  nmbrString *nmbrTmpPtr;
-  nmbrString *nmbrHypPtr;
-  nmbrString_def(bigSubstSchemeHyp);
-  nmbrString_def(bigSubstInstHyp);
-  flag unkHypFlag;
-  nmbrString_def(nmbrTmp); /* Used to force tmp stack dealloc */
-
-  if (g_Statement[statemNum].type != p_) return (4); /* Do nothing if not $p */
+char verifyProof(long statemNum) {
+  if (g_Statement[statemNum].type != p_) return 4; /* Do nothing if not $p */
 
   /* Initialize pointers to math strings in RPN stack and vs. statement. */
   /* (Must be initialized, even if severe error, to prevent crashes later.) */
-  for (i = 0; i < g_WrkProof.numSteps; i++) {
+  for (long i = 0; i < g_WrkProof.numSteps; i++) {
     g_WrkProof.mathStringPtrs[i] = NULL_NMBRSTRING;
   }
 
-  if (g_WrkProof.errorSeverity > 2) return (g_WrkProof.errorSeverity);
+  if (g_WrkProof.errorSeverity > 2) return g_WrkProof.errorSeverity;
                                     /* Error too severe to check here */
   g_WrkProof.RPNStackPtr = 0;
 
-  if (g_WrkProof.numSteps == 0) return (2);
+  if (g_WrkProof.numSteps == 0) return 2;
                         /* Empty proof caused by error found by in parseProof */
```

Comment:
not in the PR but "found by in" --> "found by" or "found in" (?)

Reply:
```suggestion
                        /* Empty proof caused by error found in parseProof */
```